src/procedures/add-ssd-existing-installation.html

```diff
@@ -23,4 +23,12 @@
 {% render "proc.include.liquid", proc: procedureOpeningCase %}
 {% render "proc.include.liquid", proc: procedureInstallNvme %}
 {% render "proc.include.liquid", proc: procedureReassembleCase %}
-{% render "proc.include.liquid", proc: procedureMoveDataDisk %}
\ No newline at end of file
+{% render "proc.include.liquid", proc: procedureMoveDataDisk %}
+
+<div class="container">
```

Comment:
I think we added such a section in more and more places no? Maybe better to add this to the layout template and introduce `related_topics` in the data section above? :thinking:

Reply:
will do. different PR.